diff --git a/node_modules/gatsby-theme-portfolio-minimal/src/components/Project/index.tsx b/node_modules/gatsby-theme-portfolio-minimal/src/components/Project/index.tsx
index b2a5385..12d9272 100644
--- a/node_modules/gatsby-theme-portfolio-minimal/src/components/Project/index.tsx
+++ b/node_modules/gatsby-theme-portfolio-minimal/src/components/Project/index.tsx
@@ -5,6 +5,7 @@ import { useMediaQuery } from '../../hooks/useMediaQuery';
 import { Icon } from '../Icon';
 import { ImageObject } from '../../types';
 import * as classes from './style.module.css';
+import htmr from 'htmr';
 
 enum LinkType {
     External = 'external',
@@ -43,7 +44,7 @@ export function Project(props: ProjectProps): React.ReactElement {
             <div className={classes.Details}>
                 <span className={classes.Category}>{props.data.category}</span>
                 <h4 className={classes.Title}>{props.data.title}</h4>
-                <p>{props.data.description}</p>
+                <p style={{textAlign: 'justify'}}>{htmr(props.data.description)}</p>
                 <div className={classes.Tags}>
                     {props.data.tags &&
                         props.data.tags.length !== 0 &&
diff --git a/node_modules/gatsby-theme-portfolio-minimal/src/sections/About/index.tsx b/node_modules/gatsby-theme-portfolio-minimal/src/sections/About/index.tsx
index 3317450..5db4df3 100644
--- a/node_modules/gatsby-theme-portfolio-minimal/src/sections/About/index.tsx
+++ b/node_modules/gatsby-theme-portfolio-minimal/src/sections/About/index.tsx
@@ -14,7 +14,7 @@ export function AboutSection(props: PageSection): React.ReactElement {
         <Animation type="fadeUp">
             <Section anchor={props.sectionId} heading={props.heading}>
                 <div className={classes.About}>
-                    <div className={classes.Description} dangerouslySetInnerHTML={{ __html: data.html }} />
+                    <div style={{textAlign: 'justify'}} className={classes.Description} dangerouslySetInnerHTML={{ __html: data.html }} />
                     <Animation type="fadeLeft" delay={200}>
                         <div className={classes.ImageWrapper}>
                             <GatsbyImage
diff --git a/node_modules/gatsby-theme-portfolio-minimal/src/sections/Contact/index.tsx b/node_modules/gatsby-theme-portfolio-minimal/src/sections/Contact/index.tsx
index 18decab..a7dc59e 100644
--- a/node_modules/gatsby-theme-portfolio-minimal/src/sections/Contact/index.tsx
+++ b/node_modules/gatsby-theme-portfolio-minimal/src/sections/Contact/index.tsx
@@ -6,6 +6,7 @@ import { SocialProfiles } from '../../components/SocialProfiles';
 import { useLocalDataSource } from './data';
 import { PageSection } from '../../types';
 import * as classes from './style.module.css';
+import htmr from 'htmr';
 
 export function ContactSection(props: PageSection): React.ReactElement {
     const response = useLocalDataSource();
@@ -14,7 +15,7 @@ export function ContactSection(props: PageSection): React.ReactElement {
     return (
         <Animation type="fadeUp">
             <Section anchor={props.sectionId} heading={props.heading} additionalClasses={[classes.Contact]}>
-                {data.description && <p className={classes.Description}>{data.description}</p>}
+                {data.description && <p style={{textAlign: 'justify'}} className={classes.Description}>{htmr(data.description)}</p>}
                 <div className={classes.Profile}>
                     {data.image.src && (
                         <GatsbyImage
diff --git a/node_modules/gatsby-theme-portfolio-minimal/src/sections/Hero/index.tsx b/node_modules/gatsby-theme-portfolio-minimal/src/sections/Hero/index.tsx
index cd193be..9bb2198 100644
--- a/node_modules/gatsby-theme-portfolio-minimal/src/sections/Hero/index.tsx
+++ b/node_modules/gatsby-theme-portfolio-minimal/src/sections/Hero/index.tsx
@@ -7,6 +7,7 @@ import { SocialProfiles } from '../../components/SocialProfiles';
 import { useLocalDataSource } from './data';
 import { PageSection } from '../../types';
 import * as classes from './style.module.css';
+import htmr from 'htmr';
 
 export function HeroSection(props: PageSection): React.ReactElement {
     const response = useLocalDataSource();
@@ -47,7 +48,7 @@ export function HeroSection(props: PageSection): React.ReactElement {
                         <u>{data.subtitle.highlight}</u>
                         {data.subtitle.suffix}
                     </h2>
-                    <p>{data.description}</p>
+                    <p style={{textAlign: 'justify'}}>{htmr(data.description)}</p>
                     <Animation type="fadeLeft" delay={600}>
                         {data.socialProfiles && (
                             <SocialProfiles from={data.socialProfiles.from} showIcon={data.socialProfiles.showIcons} />
