diff --git a/node_modules/gatsby-theme-portfolio-minimal/content/sections/stacks/stacks.example.json b/node_modules/gatsby-theme-portfolio-minimal/content/sections/stacks/stacks.example.json
new file mode 100644
index 0000000..5b1ce18
--- /dev/null
+++ b/node_modules/gatsby-theme-portfolio-minimal/content/sections/stacks/stacks.example.json
@@ -0,0 +1,17 @@
+{
+    "stacks": [
+        {
+            "label": "",
+            "image": {
+                "src": "",
+                "alt": "",
+                "objectFit": ""
+            }
+        }
+    ],
+    "button": {
+        "visible": true,
+        "label": "",
+        "initiallyShownStacks": 5
+    }
+}
diff --git a/node_modules/gatsby-theme-portfolio-minimal/src/components/Project/index.tsx b/node_modules/gatsby-theme-portfolio-minimal/src/components/Project/index.tsx
index b2a5385..12d9272 100644
--- a/node_modules/gatsby-theme-portfolio-minimal/src/components/Project/index.tsx
+++ b/node_modules/gatsby-theme-portfolio-minimal/src/components/Project/index.tsx
@@ -5,6 +5,7 @@ import { useMediaQuery } from '../../hooks/useMediaQuery';
 import { Icon } from '../Icon';
 import { ImageObject } from '../../types';
 import * as classes from './style.module.css';
+import htmr from 'htmr';
 
 enum LinkType {
     External = 'external',
@@ -43,7 +44,7 @@ export function Project(props: ProjectProps): React.ReactElement {
             <div className={classes.Details}>
                 <span className={classes.Category}>{props.data.category}</span>
                 <h4 className={classes.Title}>{props.data.title}</h4>
-                <p>{props.data.description}</p>
+                <p style={{textAlign: 'justify'}}>{htmr(props.data.description)}</p>
                 <div className={classes.Tags}>
                     {props.data.tags &&
                         props.data.tags.length !== 0 &&
diff --git a/node_modules/gatsby-theme-portfolio-minimal/src/components/Section/index.tsx b/node_modules/gatsby-theme-portfolio-minimal/src/components/Section/index.tsx
index 652340c..269f841 100644
--- a/node_modules/gatsby-theme-portfolio-minimal/src/components/Section/index.tsx
+++ b/node_modules/gatsby-theme-portfolio-minimal/src/components/Section/index.tsx
@@ -4,6 +4,8 @@ import * as classes from './style.module.css';
 interface SectionProps {
     anchor: string;
     heading?: string;
+    subHeading?: string;
+    subSubHeading?: string;
     additionalClasses?: string[];
     children: React.ReactNode;
 }
@@ -19,6 +21,8 @@ export function Section(props: SectionProps): React.ReactElement {
         <section id={props.anchor} className={classes.Section}>
             <div className={classList}>
                 {props.heading && <h3 className={classes.Heading}>{props.heading}</h3>}
+                {props.subHeading && <p className={classes.Heading}>{props.subHeading}</p>}
+                {props.subSubHeading && <p className={classes.Heading}>{props.subSubHeading}</p>}
                 {props.children}
             </div>
         </section>
diff --git a/node_modules/gatsby-theme-portfolio-minimal/src/gatsby/node/createSchemaCustomization.js b/node_modules/gatsby-theme-portfolio-minimal/src/gatsby/node/createSchemaCustomization.js
index bba16c7..52865d8 100644
--- a/node_modules/gatsby-theme-portfolio-minimal/src/gatsby/node/createSchemaCustomization.js
+++ b/node_modules/gatsby-theme-portfolio-minimal/src/gatsby/node/createSchemaCustomization.js
@@ -47,6 +47,15 @@ module.exports = ({ actions }) => {
         label: String
         initiallyShownInterests: Int
     }
+    type Stack {
+        label: String
+        image: Image
+    }
+    type StacksButton {
+        visible: Boolean
+        label: String
+        initiallyShownStacks: Int
+    }
     type ProjectLink {
         type: String
         url: String
@@ -189,6 +198,10 @@ module.exports = ({ actions }) => {
         interests: [Interest]
         button: InterestsButton
     }
+    type StacksJson implements Node @dontInfer {
+        stacks: [Stack]
+        button: StacksButton
+    }
     type ProjectsJson implements Node @dontInfer {
         projects: [Project]
         button: ProjectButton
diff --git a/node_modules/gatsby-theme-portfolio-minimal/src/gatsby/node/onPreBootstrap.js b/node_modules/gatsby-theme-portfolio-minimal/src/gatsby/node/onPreBootstrap.js
index 16773cb..96b01c7 100644
--- a/node_modules/gatsby-theme-portfolio-minimal/src/gatsby/node/onPreBootstrap.js
+++ b/node_modules/gatsby-theme-portfolio-minimal/src/gatsby/node/onPreBootstrap.js
@@ -39,6 +39,7 @@ const ContentFolder = {
     ContactSection: 'contact',
     HeroSection: 'hero',
     InterestsSection: 'interests',
+    StacksSection: 'stacks',
     LegalSection: 'legal',
     ProjectsSection: 'projects',
 };
@@ -53,6 +54,7 @@ function getContentDirectoryTree(root) {
         contactSection: path.join(root, ContentFolder.Sections, ContentFolder.ContactSection),
         heroSection: path.join(root, ContentFolder.Sections, ContentFolder.HeroSection),
         interestsSection: path.join(root, ContentFolder.Sections, ContentFolder.InterestsSection),
+        stacksSection: path.join(root, ContentFolder.Sections, ContentFolder.StacksSection),
         legalSection: path.join(root, ContentFolder.Sections, ContentFolder.LegalSection),
         projectsSection: path.join(root, ContentFolder.Sections, ContentFolder.ProjectsSection),
     };
@@ -78,6 +80,10 @@ function getExampleFiles(directory) {
             destination: path.join(directory['interestsSection'], 'interests.json'),
             example: path.join(exampleDirectory['interestsSection'], 'interests.example.json'),
         },
+        stacksSection: {
+            destination: path.join(directory['stacksSection'], 'stacks.json'),
+            example: path.join(exampleDirectory['stacksSection'], 'stacks.example.json'),
+        },
         projectsSection: {
             destination: path.join(directory['projectsSection'], 'projects.json'),
             example: path.join(exampleDirectory['projectsSection'], 'projects.example.json'),
diff --git a/node_modules/gatsby-theme-portfolio-minimal/src/sections/About/index.tsx b/node_modules/gatsby-theme-portfolio-minimal/src/sections/About/index.tsx
index 3317450..5db4df3 100644
--- a/node_modules/gatsby-theme-portfolio-minimal/src/sections/About/index.tsx
+++ b/node_modules/gatsby-theme-portfolio-minimal/src/sections/About/index.tsx
@@ -14,7 +14,7 @@ export function AboutSection(props: PageSection): React.ReactElement {
         <Animation type="fadeUp">
             <Section anchor={props.sectionId} heading={props.heading}>
                 <div className={classes.About}>
-                    <div className={classes.Description} dangerouslySetInnerHTML={{ __html: data.html }} />
+                    <div style={{textAlign: 'justify'}} className={classes.Description} dangerouslySetInnerHTML={{ __html: data.html }} />
                     <Animation type="fadeLeft" delay={200}>
                         <div className={classes.ImageWrapper}>
                             <GatsbyImage
diff --git a/node_modules/gatsby-theme-portfolio-minimal/src/sections/Contact/index.tsx b/node_modules/gatsby-theme-portfolio-minimal/src/sections/Contact/index.tsx
index 18decab..a7dc59e 100644
--- a/node_modules/gatsby-theme-portfolio-minimal/src/sections/Contact/index.tsx
+++ b/node_modules/gatsby-theme-portfolio-minimal/src/sections/Contact/index.tsx
@@ -6,6 +6,7 @@ import { SocialProfiles } from '../../components/SocialProfiles';
 import { useLocalDataSource } from './data';
 import { PageSection } from '../../types';
 import * as classes from './style.module.css';
+import htmr from 'htmr';
 
 export function ContactSection(props: PageSection): React.ReactElement {
     const response = useLocalDataSource();
@@ -14,7 +15,7 @@ export function ContactSection(props: PageSection): React.ReactElement {
     return (
         <Animation type="fadeUp">
             <Section anchor={props.sectionId} heading={props.heading} additionalClasses={[classes.Contact]}>
-                {data.description && <p className={classes.Description}>{data.description}</p>}
+                {data.description && <p style={{textAlign: 'justify'}} className={classes.Description}>{htmr(data.description)}</p>}
                 <div className={classes.Profile}>
                     {data.image.src && (
                         <GatsbyImage
diff --git a/node_modules/gatsby-theme-portfolio-minimal/src/sections/Hero/index.tsx b/node_modules/gatsby-theme-portfolio-minimal/src/sections/Hero/index.tsx
index cd193be..9bb2198 100644
--- a/node_modules/gatsby-theme-portfolio-minimal/src/sections/Hero/index.tsx
+++ b/node_modules/gatsby-theme-portfolio-minimal/src/sections/Hero/index.tsx
@@ -7,6 +7,7 @@ import { SocialProfiles } from '../../components/SocialProfiles';
 import { useLocalDataSource } from './data';
 import { PageSection } from '../../types';
 import * as classes from './style.module.css';
+import htmr from 'htmr';
 
 export function HeroSection(props: PageSection): React.ReactElement {
     const response = useLocalDataSource();
@@ -47,7 +48,7 @@ export function HeroSection(props: PageSection): React.ReactElement {
                         <u>{data.subtitle.highlight}</u>
                         {data.subtitle.suffix}
                     </h2>
-                    <p>{data.description}</p>
+                    <p style={{textAlign: 'justify'}}>{htmr(data.description)}</p>
                     <Animation type="fadeLeft" delay={600}>
                         {data.socialProfiles && (
                             <SocialProfiles from={data.socialProfiles.from} showIcon={data.socialProfiles.showIcons} />
diff --git a/node_modules/gatsby-theme-portfolio-minimal/src/sections/Interests/datastack.tsx b/node_modules/gatsby-theme-portfolio-minimal/src/sections/Interests/datastack.tsx
new file mode 100644
index 0000000..f727e1c
--- /dev/null
+++ b/node_modules/gatsby-theme-portfolio-minimal/src/sections/Interests/datastack.tsx
@@ -0,0 +1,46 @@
+import { graphql, useStaticQuery } from 'gatsby';
+import { ImageObject } from '../../types';
+
+interface StacksSectionQueryResult {
+    allStacksJson: {
+        sections: {
+            button: {
+                initiallyShownStacks: number;
+                label: string;
+                visible: boolean;
+            };
+            stacks: {
+                image: ImageObject;
+                label: string;
+            }[];
+        }[];
+    };
+}
+
+export const useStackLocalDataSource = (): StacksSectionQueryResult => {
+    return useStaticQuery(graphql`
+        query StacksSectionQuery {
+            allStacksJson {
+                sections: nodes {
+                    button {
+                        initiallyShownStacks
+                        label
+                        visible
+                    }
+                    stacks {
+                        image {
+                            alt
+                            src {
+                                childImageSharp {
+                                    gatsbyImageData(width: 20, height: 20)
+                                }
+                            }
+                            objectFit
+                        }
+                        label
+                    }
+                }
+            }
+        }
+    `);
+};
diff --git a/node_modules/gatsby-theme-portfolio-minimal/src/sections/Interests/index.tsx b/node_modules/gatsby-theme-portfolio-minimal/src/sections/Interests/index.tsx
index 7a34fd5..bdcffd0 100644
--- a/node_modules/gatsby-theme-portfolio-minimal/src/sections/Interests/index.tsx
+++ b/node_modules/gatsby-theme-portfolio-minimal/src/sections/Interests/index.tsx
@@ -5,24 +5,31 @@ import { GatsbyImage } from 'gatsby-plugin-image';
 import { Button, ButtonType } from '../../components/Button';
 import { PageSection } from '../../types';
 import { useLocalDataSource } from './data';
+import { useStackLocalDataSource } from './datastack';
 import * as classes from './style.module.css';
 
 export function InterestsSection(props: PageSection): React.ReactElement {
     const response = useLocalDataSource();
+    const responseStack = useStackLocalDataSource();
     const data = response.allInterestsJson.sections[0];
+    const dataStack = responseStack.allStacksJson.sections[0];
     const shouldShowButton = data.button.visible !== false;
     const initiallyShownInterests = data.button.initiallyShownInterests ?? 5;
     const [shownInterests, setShownInterests] = React.useState<number>(
         shouldShowButton ? initiallyShownInterests : data.interests.length,
     );
+    const [shownStacks, setShownStacks] = React.useState<number>(
+        shouldShowButton ? initiallyShownInterests : dataStack.stacks.length,
+    );
 
     function loadMoreHandler() {
         setShownInterests(data.interests.length);
+        setShownStacks(dataStack.stacks.length);
     }
 
     return (
         <Animation type="fadeUp">
-            <Section anchor={props.sectionId} heading={props.heading}>
+            <Section anchor={props.sectionId} heading={props.heading} subHeading={props.subHeading}>
                 <div className={classes.Interests}>
                     {data.interests.slice(0, shownInterests).map((interest, key) => {
                         return (
@@ -49,6 +56,33 @@ export function InterestsSection(props: PageSection): React.ReactElement {
                     )}
                 </div>
             </Section>
+            <Section anchor={props.sectionId} subSubHeading={props.subSubHeading}>
+                <div className={classes.Interests}>
+                    {dataStack.stacks.slice(0, shownStacks).map((stack, key) => {
+                        return (
+                            <Animation key={key} className={classes.Interest} type="scaleIn" delay={key * 100}>
+                                {stack.image.src && (
+                                    <GatsbyImage
+                                        image={stack.image.src.childImageSharp.gatsbyImageData}
+                                        className={classes.Icon}
+                                        alt={stack.image.alt || `Stack ${stack.label}`}
+                                    />
+                                )}{' '}
+                                {stack.label}
+                            </Animation>
+                        );
+                    })}
+                    {shouldShowButton && shownStacks < dataStack.stacks.length && (
+                        <Animation type="scaleIn" delay={(shownStacks + 1) * 100}>
+                            <Button
+                                type={ButtonType.BUTTON}
+                                onClickHandler={loadMoreHandler}
+                                label={data.button.label}
+                            />
+                        </Animation>
+                    )}
+                </div>
+            </Section>
         </Animation>
     );
 }
diff --git a/node_modules/gatsby-theme-portfolio-minimal/src/types.d.ts b/node_modules/gatsby-theme-portfolio-minimal/src/types.d.ts
index df043b8..df413af 100644
--- a/node_modules/gatsby-theme-portfolio-minimal/src/types.d.ts
+++ b/node_modules/gatsby-theme-portfolio-minimal/src/types.d.ts
@@ -20,6 +20,8 @@ interface ImageObject {
 interface PageSection {
     sectionId: string;
     heading?: string;
+    subHeading?: string;
+    subSubHeading?: string;
 }
 
 interface GatsbyNodeHelpers {
