diff --git a/node_modules/gatsby-theme-portfolio-minimal/src/components/Project/index.tsx b/node_modules/gatsby-theme-portfolio-minimal/src/components/Project/index.tsx
index b6fbc7a..65f3be4 100644
--- a/node_modules/gatsby-theme-portfolio-minimal/src/components/Project/index.tsx
+++ b/node_modules/gatsby-theme-portfolio-minimal/src/components/Project/index.tsx
@@ -43,7 +43,7 @@ export function Project(props: ProjectProps): React.ReactElement {
             <div className={classes.Details}>
                 <span className={classes.Category}>{props.data.category}</span>
                 <h4 className={classes.Title}>{props.data.title}</h4>
-                <p>{props.data.description}</p>
+                <p dangerouslySetInnerHTML={ { __html: props.data.description } }></p>
                 <div className={classes.Tags}>
                     {props.data.tags &&
                         props.data.tags.length !== 0 &&
diff --git a/node_modules/gatsby-theme-portfolio-minimal/src/sections/Articles/data.tsx b/node_modules/gatsby-theme-portfolio-minimal/src/sections/Articles/data.tsx
index 33490db..f912b69 100644
--- a/node_modules/gatsby-theme-portfolio-minimal/src/sections/Articles/data.tsx
+++ b/node_modules/gatsby-theme-portfolio-minimal/src/sections/Articles/data.tsx
@@ -50,10 +50,10 @@ export async function useMediumFeed(profileUrl: string): Promise<MediumArticle[]
 }
 
 function constructMediumFeedUrl(profileUrl: string): string {
-    const RSS_2_JSON_API = 'https://api.rss2json.com/v1/api.json?rss_url=';
+    const RSS_2_JSON_API = 'https://api.rss2json.com/v1/api.json?rss_url=https://techblog.revivaltv.id/author/dicki/rss/';
 
     // Remove trailing slashes from mediumProfile, append /feed, and make it URL friendly
-    return RSS_2_JSON_API + encodeURIComponent(profileUrl.replace(/\/+$/, '') + '/feed');
+    return RSS_2_JSON_API;
 }
 
 export const useLocalDataSource = (): ArticlePreviewQueryResult => {
diff --git a/node_modules/gatsby-theme-portfolio-minimal/src/sections/Contact/index.tsx b/node_modules/gatsby-theme-portfolio-minimal/src/sections/Contact/index.tsx
index 18decab..31b4478 100644
--- a/node_modules/gatsby-theme-portfolio-minimal/src/sections/Contact/index.tsx
+++ b/node_modules/gatsby-theme-portfolio-minimal/src/sections/Contact/index.tsx
@@ -14,7 +14,7 @@ export function ContactSection(props: PageSection): React.ReactElement {
     return (
         <Animation type="fadeUp">
             <Section anchor={props.sectionId} heading={props.heading} additionalClasses={[classes.Contact]}>
-                {data.description && <p className={classes.Description}>{data.description}</p>}
+                {data.description && <p className={classes.Description} dangerouslySetInnerHTML={ { __html: data.description } }></p>}
                 <div className={classes.Profile}>
                     {data.image.src && (
                         <GatsbyImage
diff --git a/node_modules/gatsby-theme-portfolio-minimal/src/sections/Projects/style.module.css b/node_modules/gatsby-theme-portfolio-minimal/src/sections/Projects/style.module.css
index f01bbdc..7aed5cb 100644
--- a/node_modules/gatsby-theme-portfolio-minimal/src/sections/Projects/style.module.css
+++ b/node_modules/gatsby-theme-portfolio-minimal/src/sections/Projects/style.module.css
@@ -2,6 +2,7 @@
     display: flex;
     flex-direction: row;
     margin-top: -2rem;
+    text-align: justify;
 }
 
 .MoreProjects {
