// gatsby-theme-portfolio-minimal/src/sections/Interests/datastack.tsx
import { graphql, useStaticQuery } from 'gatsby';
import { ImageObject } from '../../types';

interface StacksSectionQueryResult {
    allStacksJson: {
        sections: {
            button: {
                initiallyShownStacks: number;
                label: string;
                visible: boolean;
            };
            stacks: {
                image: ImageObject;
                label: string;
            }[];
        }[];
    };
}

export const useStackLocalDataSource = (): StacksSectionQueryResult => {
    return useStaticQuery(graphql`
        query StacksSectionQuery {
            allStacksJson {
                sections: nodes {
                    button {
                        initiallyShownStacks
                        label
                        visible
                    }
                    stacks {
                        image {
                            alt
                            src {
                                childImageSharp {
                                    gatsbyImageData(width: 20, height: 20)
                                }
                            }
                            objectFit
                        }
                        label
                    }
                }
            }
        }
    `);
};
